composer.json

```diff
@@ -14,12 +14,14 @@
         }
     ],
     "require": {
-        "ezsystems/ezpublish-kernel": "~6.0@dev || ^7.0@dev"
+        "php": "^7.1",
+        "ezsystems/ezplatform-admin-ui": "^1.5",
+        "ezsystems/ezpublish-kernel": "^7.5"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7|^6.0",
+        "phpunit/phpunit": "^6.0|^7.0",
```

Comment:
Is this "or" really necessary here? It's dev dependency, requiring either need `^6.0` or `^7.0` should be enough, right?

Reply:
> I assumed we needed it for php versions coverage, but if you are sure, then by all means suggest change.

Ok, so I've checked that and it looks like PHPUnit `^7.5` requires PHP `^7.1` which is aligned with `.travis.yml` from this PR. Requesting the latest `^7.5.16` to make composer use less memory.

```suggestion
        "phpunit/phpunit": "^7.5.16",
```